www/app/Tools/SubAgentOutputTool.php

```
@@ -0,0 +1,105 @@
+        return 'subagent:output';
+    }
+
+    public function execute(array $input, ExecutionContext $context): ToolResult
+    {
+        $taskId = trim($input['task_id'] ?? '');
+
+        if (empty($taskId)) {
+            return ToolResult::error('task_id is required');
+        }
+
+        $task = SubAgentTask::with('childConversation')->find($taskId);
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Models\SubAgentTask::with().


(staticMethod.notFound)